velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -583,4 +583,36 @@ TEST_F(BinaryFunctionsTest, toBigEndian64) {
       toBigEndian64((int64_t)-9223372036854775807 - 1));
 }
 
+TEST_F(BinaryFunctionsTest, toIEEE754Bits32) {
+  const auto toIEEE754Bits32 = [&](std::optional<float> value) {
+    return evaluateOnce<std::string, float>("to_ieee754_32(c0)", value);
+  };
+
+  EXPECT_EQ(std::nullopt, toIEEE754Bits32(std::nullopt));
```

Comment:
Please also add tests for 
std::numeric_limits<T>::infinity(), std::numeric_limits<T>::max(), std::numeric_limits<T>::min(), std::numeric_limits<T>::quiet_NaN().

Reply:
added